platform_channels/lib/src/event_channel_demo.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:platform_channels/src/accelerometer_event_channel.dart';
+
+/// Demonstrates how to use [EventChannel] to listen continuous values
+/// of Accelerometer Sensor from platform.
+///
+/// The widget uses a [StreamBuilder] to rebuild it's descendant whenever it
+/// listens a new value from the [Accelerometer.readings] stream. If there's
+/// a change in the [AccelerometerReadings.x] and [AccelerometerReadings.y],
+/// the height and width of [AnimatedContainer] are changed respectively. If
+/// there's a change in [AccelerometerReadings.z], the borderRadius is changed.
+class EventChannelDemo extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final textStyle = Theme.of(context).textTheme.headline5;
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('EventChannel Demo'),
+      ),
+      body: Center(
+        child: StreamBuilder<AccelerometerReadings>(
+          stream: Accelerometer.readings,
+          builder: (context, snapshot) {
+            if (snapshot.hasError) {
+              return Text(snapshot.error.toString());
+            } else if (snapshot.hasData) {
+              final width = 150 + snapshot.data.y * 10;
+              final height = 150 + snapshot.data.x * 10;
+              final borderRadius = snapshot.data.z;
+
+              return AnimatedContainer(
```

Comment:
The changes you've made everywhere else are solid, so I think the only remaining question is what to do about displaying the data. While the AnimatedContainer code you have here is clean, valid, Flutter, I don't think it's obvious to someone looking at the app how the container visually relates to the numbers coming out of the accelerometer (I don't have the eyes to see small changes in border radius, for example).

For now, I would consider just going back to Text widgets, so you can put your time into finishing out the rest of the app (and the plugin app). Once those are done, you'll likely have time to come back to this section of the code, and (probably) you'll have had an idea for a great way to present the data.

That's just a suggestion, though. If you want to iterate on a few designs and try to find a great one, that's cool. You might try an animated bar chart, similar to what audio equalizers do, for example.

Reply:
> 
> 
> The changes you've made everywhere else are solid, so I think the only remaining question is what to do about displaying the data. While the AnimatedContainer code you have here is clean, valid, Flutter, I don't think it's obvious to someone looking at the app how the container visually relates to the numbers coming out of the accelerometer (I don't have the eyes to see small changes in border radius, for example).

Completely makes sense to me.

> For now, I would consider just going back to Text widgets, so you can put your time into finishing out the rest of the app (and the plugin app). Once those are done, you'll likely have time to come back to this section of the code, and (probably) you'll have had an idea for a great way to present the data.

SGTM, as we finish the project we can iterate over the design. Also, if I come across any idea for displaying data, I will submitted a seperate PR for that, so we don't have our progress on hold.